region/brinstar/kraid.json

```diff
@@ -630,11 +634,23 @@
           },
           "locks": [
             {
-              "unlock": [ "Kill All Enemies" ]
+              "unlock": [ 
+                {"obstacle":{
+                  "id": "A",
+                  "requires": [ "never" ]
```

Comment:
Why never? Seeing how the reqs are already on the obstacle anyway, we could probably simplify that by putting null instead (as in, no additional requirements beyond those on the obstacle)